### PR TITLE
LPS-173196

### PR DESCRIPTION
**Pull request by styller1**
https://issues.liferay.com/browse/LPS-173196

### CODE REVIEW

**Comment by styller1** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-d2c/site-initializer-raylife-d2c-test/src/testFunctional/tests/RaylifeNavigation.testcase [hunk 1]
```diff
@@ -111,6 +123,10 @@ definition {
 	@description = "LPS-142191 - Verify if when a section has not yet been started and the user is not currently viewing this section, that section will be grayed out."
 	@priority = "5"
 	test SectionIsIncomplete {
+		property test.name.skip.portal.instance = "RaylifeNavigation#SectionIsHighlighted";
```
Comment (barrosdennys) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/site-initializer-raylife-d2c-test/src/testFunctional/tests/RaylifeNavigation.testcase:126]: @styller1 please check this one

**Comment by barrosdennys** [comment]
ci:test:sf

**Comment by styller1** [comment]
ci:test:sf

**Comment by styller1** [comment]
ci:test:sf

**Comment by barrosdennys** [comment]
ci:forward

**Pull request closed without merging**